README.md

```diff
@@ -2,126 +2,108 @@
 
   [![Stockfish][stockfish128-logo]][website-link]
 
-  <h3>Stockfish</h3>
-
-  A free and strong UCI chess engine.
-  <br>
-  <strong>[Explore Stockfish docs »][wiki-link]</strong>
-  <br>
-  <br>
-  [Report bug][issue-link]
-  ·
-  [Open a discussion][discussions-link]
-  ·
-  [Discord][discord-link]
-  ·
-  [Blog][website-blog-link]
+  # Stockfish
+
+  A free, powerful, and open-source UCI chess engine.
+  
+  **[Explore Stockfish Documentation »][wiki-link]**
+  
+  [Report Bug][issue-link] · [Open Discussion][discussions-link] · [Join Discord][discord-link] · [Read Blog][website-blog-link]
 
   [![Build][build-badge]][build-link]
   [![License][license-badge]][license-link]
-  <br>
   [![Release][release-badge]][release-link]
   [![Commits][commits-badge]][commits-link]
-  <br>
   [![Website][website-badge]][website-link]
   [![Fishtest][fishtest-badge]][fishtest-link]
   [![Discord][discord-badge]][discord-link]
 
 </div>
 
+## Table of Contents
+
+- [Overview](#overview)
+- [Key Features](#key-features)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+  - [Donating Hardware](#donating-hardware)
+  - [Improving the Code](#improving-the-code)
+- [Compiling Stockfish](#compiling-stockfish)
+- [License](#license)
+
 ## Overview
 
-[Stockfish][website-link] is a **free and strong UCI chess engine** derived from
-Glaurung 2.1 that analyzes chess positions and computes the optimal moves.
+[Stockfish][website-link] is a powerful, open-source chess engine renowned for its strength and versatility. Derived from Glaurung 2.1, Stockfish has evolved into one of the world's strongest chess engines, consistently ranking at the top of chess engine rating lists.
+
+## Key Features
+
+- Extremely strong chess analysis
+- UCI (Universal Chess Interface) compatible
+- Cross-platform support (Windows, macOS, Linux, Android, iOS)
+- NNUE (Efficiently Updatable Neural Network) evaluation
+- Extensive configuration options for advanced users
+- Active development and frequent updates
 
-Stockfish **does not include a graphical user interface** (GUI) that is required
-to display a chessboard and to make it easy to input moves. These GUIs are
-developed independently from Stockfish and are available online. **Read the
-documentation for your GUI** of choice for information about how to use
-Stockfish with it.
+## Getting Started
 
-See also the Stockfish [documentation][wiki-usage-link] for further usage help.
+### Prerequisites
 
-## Files
+Stockfish is a chess engine that requires a chess GUI (Graphical User Interface) to function as a complete chess program. Popular GUIs include:
 
-This distribution of Stockfish consists of the following files:
+- [Arena Chess GUI](http://www.playwitharena.de/) (Windows)
```

Comment:
probably not ideal to pick these three GUIs specifically. I would rather directly refer to our wiki?